### PR TITLE
Gallery example "Custom symbols": Mention own coustom symbols

### LINKED ISSUE #1349: Gallery/tutorial example showing how to use/create custom symbols
**Description of the desired feature**

I already opened a PR to add a gallery example showing different custom symbols (#1348 ). However, I feel it would be great to have a short tutorial in which it is shown how users could generate their own custom symbols (based on the information and instructions available from https://docs.generic-mapping-tools.org/latest/cookbook/custom-symbols.html).

### PR DESCRIPTION
**Pull request by yvonnefroehlich**
**Description of proposed changes**

This PR adds a comment to the already existing gallery example [Custom symbols](https://www.pygmt.org/dev/gallery/symbols/custom_symbols.html) regarding own custom symbols.

Actually, issue #1349 suggests adding a standalone gallery example or tutorial. However, the macro language that is used to define own custom symbols is quite complicated, there is no difference to GMT, and the GMT documentation is quite detailed (please see https://docs.generic-mapping-tools.org/6.5/reference/custom-symbols.html#the-macro-language). Furthermore, plotting own custom symbols works similar to plotting built-in custom symbols (except including the `.def` file extension). So, mentioning the possibility of creating own custom symbols and the reference to the GMT documentation in the PyGMT gallery example [Custom symbols](https://www.pygmt.org/dev/gallery/symbols/custom_symbols.html) may be enough.

**Preview**: https://pygmt-dev--3186.org.readthedocs.build/en/3186/gallery/symbols/custom_symbols.html

Fixes #1349 (partly)

**Reminders**

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
- [ ] Write detailed docstrings for all functions/methods.
- [ ] If wrapping a new module, open a 'Wrap new GMT module' issue and submit reasonably-sized PRs.
- [ ] If adding new functionality, add an example to docstrings or tutorials.
- [ ] Use underscores (not hyphens) in names of Python files and directories.

**Slash Commands**

You can write slash commands (`/command`) in the first line of a comment to perform
specific operations. Supported slash command is:

- `/format`: automatically format and lint the code

### CODE REVIEW

**Review by seisman — APPROVED** [review]

File: examples/gallery/symbols/custom_symbols.py [hunk 1]
```diff
@@ -4,10 +4,21 @@
...
-Technical Reference :gmt-docs:`reference/custom-symbols.html`.
+the ``style`` parameter.
+
+In total 41 custom symbols are already included of which the following plot
+shows five exemplary ones. The symbols are shown underneath their
+corresponding names. For the remaining symbols see the GMT Technical
+Reference :gmt-docs:`reference/custom-symbols.html`.
+
+Beside these built-in custom symbols GMT allows users to define their own
+custom symbols. For this a specific macro language is used. An detailed
```
Comment (michaelgrund) [inline examples/gallery/symbols/custom_symbols.py:15]:
```suggestion
custom symbols. For this, a specific macro language is used. An detailed
```

Comment (michaelgrund) [inline examples/gallery/symbols/custom_symbols.py:15]:
```suggestion
custom symbols. For this a specific macro language is used. A detailed
```

Comment (yvonnefroehlich) [reply examples/gallery/symbols/custom_symbols.py]: See commit https://github.com/GenericMappingTools/pygmt/pull/3186/commits/2e0472e67be8a7e947f1e907fa7ca89128f24ce1.

Comment (yvonnefroehlich) [reply examples/gallery/symbols/custom_symbols.py]: Included in https://github.com/GenericMappingTools/pygmt/pull/3186/commits/3638881fc34bfe785df0921e201cbb93a470e07d.

**Review by michaelgrund — APPROVED** [review]

**Pull request merged**